lib/OpenQA/WebAPI.pm

```diff
@@ -208,6 +208,8 @@ sub startup {
     $pub_admin_r->get('/workers/:worker_id')->name('admin_worker_show')->to('workers#show');
     $pub_admin_r->get('/workers/:worker_id/ajax')->name('admin_worker_previous_jobs_ajax')
       ->to('workers#previous_jobs_ajax');
+    $pub_admin_r->delete('/workers/:worker_id')->name('admin_worker_delete')
```

Comment:
It would be good if you change it in the way @kraih wants to change all of those lines: https://github.com/os-autoinst/openQA/pull/2030

Reply:
Right, that would be `$pub_admin_r->delete('/workers/<worker_id:num>')`. The feature is explained [here](https://mojolicious.org/perldoc/Mojolicious/Guides/Routing#Placeholder-types). Basically just a name for a regex.